### DISCUSSION TITLE
Regrading playwright tests triggered in github action

### DISCUSSION THREAD

**Discussion by ros-fadv** [community/community#75044] [Actions]
### Select Topic Area

Question

### Body

When playwright test for web application is triggered in github action the api data is not getting fetched, In trace  noticed in network getting status as -1 for method POST , but when tests are executed locally it is successful.

**Comment by ev1sl** [comment]
The most likely cause of the issue is differences in network access or configuration between your local environment and the GitHub Actions runners, potentially leading to the API being unreachable or behaving differently in the CI environment.
